rs-commerce/src/components/login-page-btn/login-page-btn.tsx

```diff
@@ -0,0 +1,30 @@
+import { LOGIN_PAGE_TEXT } from 'constants/constants';
+import { useAppSelector } from 'hooks/typed-react-redux-hooks';
+import authSelector from 'redux/selectors';
+import ButtonBig from 'components/button-big/button-big';
+import style from './style.module.css';
+
+function LoginPageBtn(): JSX.Element {
+  const { emailValue, isEmailValid, passwordValue, isPasswordValid } = useAppSelector(authSelector);
+
+  const handleBtnLogin = (): void => {
+    if (isEmailValid && isPasswordValid) {
+      console.log('handle login with data: ', { email: emailValue, password: passwordValue });
+    }
+  };
+
+  const loginPageBtnStyle = (): string =>
+    isEmailValid && isPasswordValid
+      ? `${style['login-page-btn']}`
+      : `${style['login-page-btn']} ${style['btn-not-active']}`;
+
+  return (
+    <ButtonBig
+      style={loginPageBtnStyle()}
+      content={LOGIN_PAGE_TEXT.loginPageBtn}
+      onClick={handleBtnLogin}
+    />
```

Comment:
In case of using `children` this component can pass text or JSX as usual tag with content inside `<ButtonBig>{LOGIN_PAGE_TEXT.loginPageBtn}</ButtonBig>`
With text the benefits are not very bright, but for JSX it is very convenient. Just proposal, in this case maybe prop content is ok. Read about [here](https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children)